usr/local/www/services_dhcp_relay.php

```diff
@@ -106,106 +125,73 @@
 $shortcut_section = "dhcp";
 include("head.inc");
 
-?>
-
-<script type="text/javascript">
-//<![CDATA[
-function enable_change(enable_over) {
-	if (document.iform.enable.checked || enable_over) {
-		document.iform.server.disabled = 0;
-		document.iform.interface.disabled = 0;
-		document.iform.agentoption.disabled = 0;
-	} else {
-		document.iform.server.disabled = 1;
-		document.iform.interface.disabled = 1;
-		document.iform.agentoption.disabled = 1;
-	}
+if ($dhcpd_enabled)
+{
+	echo "DHCP Server is currently enabled. Cannot enable the DHCP Relay service while the DHCP Server is enabled on any interface.";
```

Comment:
I was thinking about it, wrapped in in the print_error_messages first. Maybe the messaging system can be abstracted.